prometheus_client/openmetrics/parser.py

```diff
@@ -364,6 +336,31 @@ def _parse_nh_struct(text):
         pos_deltas=pos_deltas,
         neg_deltas=neg_deltas
     )
+  
+
+def _compose_spans(span_matches, spans_name):
```

Comment:
nit: might want to add a comment what this does, because the list comprehension takes a minute to understand :)

Reply:
Yes, that makes sense! I've also added comments to the other less complicated function, while I was at it :D